src/commands/retrieve.js

```diff
@@ -0,0 +1,24 @@
+const Discord = require('discord.js');
+const logger = require('./../logger');
+const db = require('./../mongo');
+
+module.exports = {
+	retrieve : {
+		name : 'retrieve',
+		args : true,
+		description : 'Retrieve a command from the database.',
+		async execute(message, data) {
```

Comment:
https://stackoverflow.com/questions/8902687/javascript-storing-function-in-object-bad-practice 

Following the first method mentioned in this article, you'll also get to use es6 syntax for the functions...
e.g. `execute: (message,data) {`

Reply:
e.g. ‘execute: (message,data) => {‘
Oops correction

Reply:
e.g. ‘execute: (message,data) => {‘
Oops correction